impacket/krb5/kerberosv5.py

```diff
@@ -345,7 +349,10 @@ def getKerberosTGT(clientName, password, domain, lmhash, nthash, aesKey='', kdcH
         # probably bad password if preauth is disabled
         if preAuth is False:
             error_msg = "failed to decrypt session key: %s" % str(e)
-            raise SessionKeyDecryptionError(error_msg, asRep, cipher, key, cipherText)
+            # Commenting error below in order to return tgt, so that Kerberoast through AS-REQ can be conducted
```

Comment:
don't remove Exception raising as client code might be expecting this exception to be thrown. Contemplate kerbroasting by adding  a new parameter in getKerberosTGT() and then avoid raising the exception only in that scenario.

Reply:
Not sure I understand how you want it implemented, can you etiher di the change or write in this thread what you had in mind?